README.md

```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
 
 ## Getting started
 
-To run the server, you'll need [Node.js](https://nodejs.org/download/). You'll also need [nvm](https://github.com/creationix/nvm).
+### Dependencies / Tooling
+Before you start, ensure you have the following installed:
+
+- [node](https://nodejs.org/en/download/package-manager/#osx)
+- [nvm](https://github.com/creationix/nvm#installation) or [n](https://github.com/tj/n#installation)
+- [yarn](https://yarnpkg.com/)
+- [Postgres](https://www.postgresql.org/)
+- [Docker](https://docs.docker.com/engine/installation/mac/)
+- [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+- [AWS CLI](https://aws.amazon.com/cli/)
```

Comment:
don't really need this one actually

Reply:
Referring to AWC CLI being removed here @jseppi ?

Reply:
yup!